terraform/lambda-role.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Are the files under `terraform` intentional or should those be gone now that `src/terraform` exists?

Reply:
I intentionally have an additional one at the root as a template to create for other demos.